SubFinder.py

```diff
@@ -48,6 +49,14 @@ def __str__(self):
               'maximum threads is {}, you specify {}'
         return msg.format(self.max_threads, self.threads)
 
+def isVideoFile(file):
+    if os.path.isfile(file):
+        types = mimetypes.guess_type(file)
+        mtype = types[0]
+        if mtype and mtype.split('/')[0] == 'video' or os.path.splitext(file)[1] in VIDEO_EXTS:
```

Comment:
well, **_and_** has higher precedence than **_or_**, which is a basic principle for almost all program languages, isn't it?